tests/test_docker_compose_edge_cases.py

```
@@ -1,6 +1,4 @@
-"""
-Edge cases and boundary condition tests for Qdrant Docker Compose configuration
-"""
+"""Edge cases and boundary condition tests for Qdrant Docker Compose configuration."""
```

Ruff (0.12.2)

File `tests/test_docker_compose_edge_cases.py` is part of an implicit namespace package. Add an `__init__.py`.

(INP001)